compiler/test/stdlib/buffer.test.gr

```diff
@@ -207,7 +207,7 @@ let slices = [>
   ("0123456789", -1, -1),
   ("0123456789", 2, 3),
   ("0123456789", 0, 10),
-  ("0123456789", 6, 6),
+  ("0123456789", 6, 6),,
```

Comment:
this has two trailing commas

Reply:
Yes, bad push, those tests are cleaned up again now